submissions/smillims/js-dom/script.js

```diff
@@ -53,16 +50,16 @@ function displayFact(navLink) {
 };
 
 
-document.addEventListener("DOMContentLoaded", function(e) {
+document.addEventListener("DOMContentLoaded", function(el) {
 	createNavItem();
-	ul.addEventListener('click', function(e) {
-	  let target = e.target;
-	  e.preventDefault();
+	ul.addEventListener('click', function(el) {
+	  let target = el.target;
+	  el.preventDefault();
 	  const navItemActive = document.querySelector('.list-item.active');
 	  if (navItemActive) {
 		 navItemActive.classList.remove('active');
 	  }
-	  target.parentElement.classList.add('active');
+	  target.closest('.list').classList.add('active');
```

Comment:
You added active class to `ul`, is it ok ?
`<ul class="list active">`

Reply:
fixed (i think)